contracts/DelayModule.sol

```diff
@@ -44,17 +44,27 @@ contract DelayModule {
   // Mapping of approved modules
   mapping(address => bool) public modules;
 
+  bool public isInitialized = false;
+
+  constructor() {
+    isInitialized = true;
```

Comment:
Why is `isInitialized` set to `true` in the constructor?
Won't this make it always fail the check on line 58? `require(!isInitialized, "Module is already initialized");`